CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Zipkin support added for tracing Knative function call requests. More info [here](./docs/developers_guide.md#Knative-request-tracing)
 - added support for MinIO object store. More info [here](./docs/developers_guide.md#MinIO-S3-service)
 - Added an automated tail-latency-aware profiler that collects the metrics for [TopDown](https://ieeexplore.ieee.org/document/6844459) characterization from Intel.
+- [alpha] Added knative eventing support.
```

Comment:
Please specify that only their in-memory channel is supported as of now and through integration tests are missing (unless you plan to add an eventing example in this PR). Maybe adding an invocation without validation in this PR is a good idea if it is more or less trivial. If it would take more than 30min, let's postpone it to another PR.